src/components/AppliedFiltersDisplay.tsx

```diff
@@ -46,20 +46,20 @@ export function AppliedFiltersDisplay(props: AppliedFiltersDisplayProps): JSX.El
     hierarchicalFacetsDelimiter = DEFAULT_HIERARCHICAL_DELIMITER,
     cssClasses = {}
   } = props;
-  const answersActions = useAnswersActions();
-  const isVertical = useAnswersState(state => state.meta.searchType) === SearchTypeEnum.Vertical;
+  const SearchActions = useSearchActions();
```

Comment:
nit: can this be lowercase like it used to be (`searchActions`)? and similarly in other files

Reply:
whoops, thanks! I'll change all of them back, I just used the replace function lol